src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/TransitLayer.java

```diff
@@ -17,6 +17,16 @@
 import org.opentripplanner.transit.model.site.StopLocation;
 import org.opentripplanner.transit.service.StopModel;
 
+/**
+ * This is a replica of public transportation data already present in TransitModel, but rearranged
+ * and indexed differently for efficient use by the Raptor router. Patterns and trips are split out
+ * by days, retaining only the services actually running on any particular day.
+ *
+ * TODO RT_AB: this name is apparently modeled after R5, where the TransportNetwork is split into
+ *  two layers (one for the streets and one for the public transit data). Here the situation is
+ *  different: this seems to be an indexed and rearranged copy of the main transit data (a
+ *  TransitModel instance).
+ */
```

Comment:
The main point here was that there are two terms (TransitModel and TransitLayer) that imply they're two different things, but it seems like they might be fragmented pieces of the same thing. I was wondering if there's some reason one of them is singled out as a "Layer" or if they should just be combined.

The term "layer" in R5 was inspired by geographic information systems (GIS) terminology where a "layer" is a set of geolocated data that is typically overlaid visually on other such layers. A street network might be in one layer, and the transit network in another layer on the same map. It's not necessarily the right term but it reflects organizing the transportation network into two independent networks that are largely separate from one another but connected at specific points. I did not expect the term Layer to appear with this meaning in OTP, unless it was intentionally adopting R5 terminology and fully replacing some other term like TransitNetwork, TransitGraph, or TransitModel.

Personally I find it disorienting to use the word "Model" to refer to something that has been instantiated (a graph of object instances), because I think of the Model as the abstract structure that has been translated into classes. Analogous to the data model behind a database, as contrasted with the specific rows of data that have populated the tables in a particular physical instance of that database. In this view, a class that serves as an entry point or index into an instantiated representation of a transportation network is not "a model", but makes use of _instances_ of the model. In other words, I immediately understand when I see a package or base class called Model but I'm a little thrown off by concrete classes and instances being referred to as Model. 

But I recognize that I may be distorting / misinterpreting OOP terminology as used in a certain community of practice, so I will read up on this.

Reply:
>  We should aim for transit-model and street-model. Should we rename this class?

To directly answer your question: if we renamed it according to your preferred standard terminology, I think it would be called TransitModel, and there's already a TransitModel. So the question is more like: is this just part of TransitModel? I think you've answered this by saying your long-term expectation is to merge them. I have updated the comment accordingly.

Reply:
After thinking about this a bit, maybe this is a simpler/better version of my comment about Model: The "transit model" is the whole collection of classes that model transit. If there's a class called TransitModel, then the term "transit model" would mean both this one specific class TransitModel, and the collection of classes this one specific class belongs to: the class TransitModel is one of many classes that make up the transit model. This seems confusing to me - is this a conventional practice? We can discuss briefly sometime outside this PR.

Reply:
Thanks for the references and details. These points fit with my understanding of what the model is. The remaining question for me is about there being one specific concrete, instantiable class called Model, which is itself part of the model. But I think that question is not essential for this PR will be easier to resolve in a meeting discussion.